src/pal/src/arch/arm/context2.S

```diff
@@ -193,8 +193,13 @@ LEAF_ENTRY RtlRestoreContext, _TEXT
     ldr R2, [r0, #(CONTEXT_Cpsr)]
     msr APSR, r2    
     
-    add r0, CONTEXT_R0
-    ldmia r0, {r0-r12, sp, lr, pc}
+    mov r12, r0 // ideally we would ldmia r0, {r0-r12, sp, lr, pc} here, but that isn't supported on new clang
+    add r12, CONTEXT_R0 // so we'll burn r12 as the IPC reg for now -- TODO: is this ok?
+    ldmia r12, {r0-r11}
```

Comment:
ldmia is increment after. It should be ldm without increment, or the next line needs to take the increment into account.